cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc

```diff
@@ -0,0 +1,14 @@
+[id="cnv-viewing-ip-of-vm-vnic"]
+= Viewing the IP address of vNICs on a virtual machine
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-viewing-ip-of-vm-vnic
+toc::[]
+
+The QEMU guest agent runs on the virtual machine and passes the IP address of attached vNICs to the host, allowing you to view it from both the web console and the `oc` client.
```

Comment:
What is `it`?

Reply:
Mike Patton would be proud :)
But seriously, I've updated with s/it/the IP address/'

Reply:
We care a lot...